src/Contracts/Services/Warehouse/Projection.cs

```diff
@@ -5,7 +5,29 @@ namespace Contracts.Services.Warehouse;
 
 public static class Projection
 {
-    public record Inventory(Guid Id, Guid OwnerId, bool IsDeleted) : IProjection;
+    public record Inventory(Guid Id, Guid OwnerId, bool IsDeleted) : IProjection
+    {
+        public static implicit operator Protobuf.InventoryGridItem(Inventory inventory)
+            => new()
+            {
+                InventoryId = inventory.Id.ToString(),
+                OwnerId = inventory.OwnerId.ToString(),
+                IsDeleted = inventory.IsDeleted
+            };
+    }
 
-    public record InventoryItem(Guid Id, Guid InventoryId, Dto.Product Product, int Quantity, string Sku, bool IsDeleted) : IProjection;
+    public record InventoryItem(Guid Id, Guid InventoryId, Dto.Product Product, int Quantity, string Sku,
+        bool IsDeleted) : IProjection
+    {
+        public static implicit operator Protobuf.InventoryItemListItem(InventoryItem inventoryItem)
+            => new()
+            {
+                Id = inventoryItem.Id.ToString(),
+                InventoryId = inventoryItem.InventoryId.ToString(),
+                IsDeleted = inventoryItem.IsDeleted,
```

Comment:
Remove IsDeleted from proto. It's an internal concern

Reply:
ok